PSReadLine/Render.Helper.cs

```diff
@@ -0,0 +1,161 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+
+namespace Microsoft.PowerShell
+{
+    public partial class PSConsoleReadLine
+    {
+        private void WriteBlankLines(int count)
+        {
+            _console.BlankRestOfLine();
+            for (int i = 1; i < count; i++)
+            {
+                _console.Write("\n");
```

Comment:
Environment.Newline?

Reply:
PSReadLine always uses `\n` for rendering. The only exception is in `CaptureScreen`, which copies the buffer cells on Windows  platform.